### PR TITLE
xbps-src: expand note about -t flag.

### PR DESCRIPTION
**Pull request by ericonr**
Related to #24314

### CODE REVIEW

**Comment by q66** [comment]
afaik there is work in progress to support `renameat2` in zfs, so this might not be true after a while

**Comment by ericonr** [comment]
Made it more generic, then.

**Comment by q66** [comment]
the wording makes it sound like they are two separate things, i'd reword it it a bit, something like "This flag requires xbps-uchroot(1), so it won't work..."

**Comment by ericonr** [comment]
I'm not sure it's fully related... If you set `XBPS_CHROOT_CMD=uchroot` in `etc/conf`, things still work just fine, unless you also add `-t` to the `xbps-src` command line.

**Comment by q66** [comment]
hm, you're right i guess...
